stubs/pyinstaller/METADATA.toml

```
@@ -1,4 +1,4 @@
-version = "6.1.*"
+version = "6.2.*"
 upstream_repository = "https://github.com/pyinstaller/pyinstaller"
 requires = ["types-setuptools"]
 requires_python = ">=3.8"
```

If we did this, we wouldn't need the new allowlist entry (I think), since this would mean that `argcomplete` would be installed in CI, which would mean we'd never hit the `except ImportError` branch [here](https://github.com/pyinstaller/pyinstaller/blob/dd40fb2cd75b86b5bbe23843bafde09740f451af/PyInstaller/__main__.py#L27-L32), which would mean stubtest would be able to detect that it's just imported from another module

```suggestion
requires_python = ">=3.8"

[tool.stubtest]
extras = ["completion"]
```

---

Makes sense, let's try that. I'll add a comment specifying why though, since the extra wouldn't affect the end-user's type hints